apis/Google.Storage.V1/Google.Storage.V1.IntegrationTests/StorageFixture.cs

```diff
@@ -97,9 +97,9 @@ public sealed class StorageFixture : IDisposable, ICollectionFixture<StorageFixt
         public HttpClient HttpClient { get; } = new HttpClient();
 
         /// <summary>
-        /// Gets the path to a service account credentials JSON file.
+        /// Gets a <see cref="UrlSigner"/> instance which can be used for testing.
         /// </summary>
-        public string CredentialsFilePath { get; } = GetCredentialsFilePath();
+        public UrlSigner UrlSigner { get; } = new UrlSigner(GetCredentialsFilePath());
```

Comment:
I was under the impression that this would initialize the read only property once (which was the intended behavior). I've never actually used this pattern before, but when debugging it just now, it seems it is functioning as I'd intended.